config/config-windows.yaml

```diff
@@ -284,7 +370,18 @@ service:
         - filter/cloud-monitoring
         # add additional custom processors below this line if desired
         - resourcedetection
-      exporters:        
+      exporters:
+        - stackdriver
+
+    # reports agent self-observability metrics to cloud monitoring
+    metrics/agent:
+      receivers:
+        - prometheus/agent
+      processors:
+        - filter/agent
+        - metricstransform/agent
+        - resourcedetection
+      exporters:
         - stackdriver
```

Comment:
nit: It may make sense to keep this exporter separate from the other pipeline for isolation purposes